### PR TITLE
Added loadout: UK_Army_2005

### PR DESCRIPTION
**Pull request by AChesheireCat**
Round 2, fight!
Fix'd engie backpack class name error
Removed 4x scopes on normal infantry, gave to SL
Fix'd various dependencies

### CODE REVIEW

**Comment by AChesheireCat** [comment]
Has FIR stuff btw, so on hold like the Rangers_2020 loadout.

File: addons/tmf_loadouts/loadouts/UK_Army_2005.hpp [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_Pilot"};
+	headgear[] = {
+		"CFP_SOARCREW_BLUE",
+		"CFP_SOARCREW_STACHE",
+		"CFP_SOARCREW_RED",
+		"CFP_SOARCREW_SKULL",
+		"H_CrewHelmetHeli_B",
+		"FIR_HGU56P_Skull",
+		"FIR_HGU56P_Skull_Death",
+		"MNP_Helmet_Dslyecxi"
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/UK_Army_2005.hpp:473]: Unlisted dependency

File: addons/tmf_loadouts/loadouts/UK_Army_2005.hpp [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+class eng : car {
+	displayName = "Combat Engineer (Explosives)";
+	traits[] = {"engineer", "explosiveSpecialist"};
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_Engineer"};
+	backpack[] = {"CUP_B_ACRPara_dpm"};
+	sidearmWeapon[] = {"ACE_VMH3"};
+	items[] += {
+		"ACE_DefusalKit",
+		"ACE_EntrenchingTool",
+		"ACE_Clacker"
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/UK_Army_2005.hpp:518]: Wirecutters?

File: addons/tmf_loadouts/loadouts/UK_Army_2005.hpp [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+class engm : car {
+	displayName = "Combat Engineer (Mines)";
+	traits[] = {"engineer", "explosiveSpecialist"};
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_Engineer"};
+	backpack[] = {"CUP_B_ACRPara_dpm"};
+	sidearmWeapon[] = {"ACE_VMM3"};
+	items[] += {
+		"ACE_DefusalKit",
+		"ACE_EntrenchingTool",
+		"ACE_Clacker"
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/UK_Army_2005.hpp:537]: Wirecutters?

**Review by Freddo3000 — CHANGES_REQUESTED** [review]
On hold until FIR is clarified

File: addons/tmf_loadouts/loadouts/UK_Army_2005.hpp [hunk 4]
```diff
@@ -0,0 +1,553 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+//Dependencies:
+//		* CUP Uniforms
+//		* CUP Weapons
+//		* Community Factions Project
+// 	* NIArms P226
+// 	* USM
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/UK_Army_2005.hpp:9]: Differentiating amount whitespace, pls fix.

File: addons/tmf_loadouts/loadouts/UK_Army_2005.hpp [hunk 5]
```diff
@@ -0,0 +1,553 @@
...
+		LIST_2("CUP_HandGrenade_L109A1_HE"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_Grenadier"};
+	primaryWeapon[] = {"CUP_arifle_L85A2_GL"};
+	scope[] = {};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/UK_Army_2005.hpp:93]: Not needed line

File: addons/tmf_loadouts/loadouts/UK_Army_2005.hpp [hunk 6]
```diff
@@ -0,0 +1,553 @@
...
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backPack[] = {"Redd_Milan_Static_Barrel"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/UK_Army_2005.hpp:318]: Unlisted dependency, Redd'n'Tank

File: addons/tmf_loadouts/loadouts/UK_Army_2005.hpp (hunk 1 above)
Comment (Freddo3000) [reply addons/tmf_loadouts/loadouts/UK_Army_2005.hpp:470]: MNP is still not listed

**Review by Freddo3000 — CHANGES_REQUESTED** [review]

File: addons/tmf_loadouts/loadouts/UK_Army_2005.hpp [hunk 7]
```diff
@@ -218,7 +218,7 @@ class dm : r {
 	displayName = "Designated Marksman";
 	primaryWeapon[] = {"CUP_arifle_L86A2"};
 	scope[] = {"cup_optic_susat"};
-	secondaryWeapon[] = {"hlc_pistol_P226US"};
+	secondaryWeapon[] = {"hlc_pistol_P226US"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/UK_Army_2005.hpp:221]: sidearmWeapon

**Review by Freddo3000 — CHANGES_REQUESTED** [review]

File: addons/tmf_loadouts/loadouts/UK_Army_2005.hpp (hunk 7 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/UK_Army_2005.hpp:221]: Uhhh, fix'd I think? No clue how that happened.

**Review by Freddo3000 — APPROVED** [review]

**Pull request merged**